### PR TITLE
fix(a11y): Update OpenDyslexic from 0.920 to 2.001

### PR DESCRIPTION
**Pull request by nickvergessen**
- Ref https://github.com/nextcloud/server/issues/39278

## Summary

Normal font | OpenDyslexic v0.920 | OpenDyslexic v2.001
---|---|---
Bildschirmfoto vom 2024-09-12 16-39-55 | Bildschirmfoto vom 2024-09-12 16-39-20 | Bildschirmfoto vom 2024-09-12 16-39-38

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/theming/lib/Themes/DyslexiaFont.php [hunk 1]
```diff
@@ -43,30 +43,22 @@ public function getCSSVariables(): array {
 	}
 
 	public function getCustomCss(): string {
-		$fontPathWoff = $this->urlGenerator->linkTo('theming', 'fonts/OpenDyslexic-Regular.woff');
```
Comment (nickvergessen) [inline apps/theming/lib/Themes/DyslexiaFont.php:46]: otf is supported by all supported browsers (IE was the last one), so we can drop the others

**Review by jancborchardt — APPROVED** [review]
Wow, amazing improvement! FYI @xBelladonna if you want to check it out. :)

File: apps/theming/lib/Themes/DyslexiaFont.php (hunk 1 above)
Comment (solracsf) [reply apps/theming/lib/Themes/DyslexiaFont.php:46]: Beside browser support, WOFF is generaly more lightweight too.

image

Comment (nickvergessen) [reply apps/theming/lib/Themes/DyslexiaFont.php:46]: But WOFF is not provided by the creator and I don't know what happens with licenses when using a convertor of any kind `¯\_(ツ)_/¯` The files are ~41KB and ~42KB. I really don't care too much about such a little size to risk license issues

Comment (nickvergessen) [reply apps/theming/lib/Themes/DyslexiaFont.php:46]: That being said, the old font file was ~250KB and the WOFF was ~120KB there is made a bit more sense

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
I think this should be backported

**Pull request merged**